ip-geo.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Health Bot Geo Lookup</title>
+</head>
+
+<body>
+    <h1>Health bot</h1>
+    <p>Hello <span id="user_ip"></span> from <span id="user_countrycode"></span> <span id="user_region"></span></p>
+    <p>The health bot is <span id="user_healthbotenabled"></span> for your country and state.</p>
+    <div id="disabled_message"></div>
+
+    <script type="application/javascript">
+    
+        /* this function is automatically invoked via the geojs script and json object is populated with location information */
+        function geoip(json) {
+            determineBotEnabled(json);
+        }
+
+        /* this function uses ip location information to determine if the health bot should be enabled */        
+        function determineBotEnabled(json) {
+
+            // update page elements with information collected from geojs ip location lookup
+            var userip = document.getElementById("user_ip");
+            var countrycode = document.getElementById("user_countrycode");
+            var region = document.getElementById("user_region");
+            var healthBotEnabled = document.getElementById("user_healthbotenabled");
+            userip.textContent = json.ip;
```

Comment:
Should we check if these properties exist because I believe when we try to access them and they don't we may end up just throwing.